data/json/itemgroups/Clothing_Gear/costumes.json

```diff
@@ -334,13 +334,7 @@
       { "item": "bodysuit_lycra", "prob": 25 },
       { "item": "suit", "prob": 10 },
       { "item": "tux", "prob": 10 },
-      { "item": "wolfsuit", "prob": 10 },
-      { "item": "clown_suit", "prob": 10 },
-      { "item": "dragonsuit", "prob": 10 },
-      { "item": "felinesuit", "prob": 10 },
-      { "item": "bondage_suit", "prob": 15 },
```

Comment:
accidentally removed the clown suit and bondage suit. that being said, why is the bondage suit in this pool to begin with?